security/pkg/k8s/configutil.go

```diff
@@ -51,7 +51,8 @@ func InsertDataToConfigMap(client corev1.ConfigMapsGetter, lister listerv1.Confi
 			// The lister may be outdated, thinking there's no config map when there is, so the attempt to create one will fail
 			// If this is the case, try updating it instead
 			if errors.IsAlreadyExists(err) {
-				return UpdateDataInConfigMap(client, configmap, data)
+				_, err := client.ConfigMaps(meta.Namespace).Update(context.TODO(), configmap, metav1.UpdateOptions{})
+				return err
```

Comment:
It would probably be good to explain the error, like it's explained elsewhere in this function (e.g. `return fmt.Errorf("error when updating configmap %v: %v", meta.Name, err)`) to keep things consistent.

Reply:
done